build.sbt

```diff
@@ -33,7 +33,7 @@ lazy val global = root
       settings,
       publish / skip := true
   )
-  .disablePlugins(AssemblyPlugin)
+//  .disablePlugins(AssemblyPlugin)
```

Comment:
This line wasn't changed recently, but why was it set this way? (Getting rid of it doesn't seem to help with the current issue.) @jdidion

Reply:
You need to skip assembly on the global project or you'll get a basically empty JAR created for the root project.

Reply:
Aha, thanks @jdidion